nixos/modules/services/x11/clight.nix

```diff
@@ -0,0 +1,112 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.clight;
+
+  toConf = v:
+    if builtins.isFloat v then toString v
+    else if true == v     then "true"
+    else if false == v    then "false"
```

Comment:
You can combine these into `if isBool v then boolToString v`

Reply:
Done.